src/main/java/edu/harvard/iq/dataverse/dataverse/banners/dto/BannerMapper.java

```
@@ -0,0 +1,129 @@
+
+        dto.getDataverseLocalizedBanner().forEach(fuDto -> {
+            DataverseLocalizedBanner dataverseLocalizedBanner = new DataverseLocalizedBanner();
+            dataverseLocalizedBanner.setImage(fuDto.getFile().getContents());
+            dataverseLocalizedBanner.setImageLink(fuDto.getImageLink());
+            dataverseLocalizedBanner.setDataverseBanner(banner);
+            dataverseLocalizedBanner.setLocale(fuDto.getLocale());
+            dataverseLocalizedBanner.setContentType(fuDto.getFile().getContentType());
+            dataverseLocalizedBanner.setImageName(fuDto.getFile().getFileName());
+
+            banner.getDataverseLocalizedBanner().add(dataverseLocalizedBanner);
+        });
+        return banner;
+    }
+
+    public DataverseBannerDto mapToNewBanner(Long dataverseId) {
+        DataverseBannerDto dto = new DataverseBannerDto();
+
+        dto.setDataverseId(dataverseId);
+        dto.setDataverseLocalizedBanner(mapDefaultLocales());
+
+        return dto;
+    }
+
+    private List<DataverseLocalizedBannerDto> mapDefaultLocales() {
+        Map<String, String> locales = new DataverseLocaleBean().getDataverseLocales();
+
+        return locales.entrySet().stream()
+                .map(e -> new DataverseLocalizedBannerDto(e.getKey(), e.getValue()))
+                .collect(Collectors.toList());
+    }
+
+    private ByteArrayOutputStream convertImageToMiniSize(byte[] image) {
+
+        try {
+            BufferedImage loadedImage = ImageIO.read(new ByteArrayInputStream(image));
+
+            BufferedImage resizedImage = Scalr.resize(loadedImage,
+                    BannerLimits.MAX_WIDTH.getValue() / 3,
+                    BannerLimits.MAX_HEIGHT.getValue() / 3);
+
+            ByteArrayOutputStream os = new ByteArrayOutputStream();
+            ImageIO.write(resizedImage, "jpeg", os);
+            return os;
+
+        } catch (IOException e) {
+            logger.fine("There was a problem loading the image");
```

We really want to almost ignore this kind of situations?
That way we could possibly never discover that something is wrong with app when running on production.
I think that it would be better to let the app crash.
```
} catch (IOException e) {
    throw new RuntimeException("There was a problem loading the image", e);
}
```